src/app/shared/pipes/capitalize.pipe.ts

```diff
@@ -0,0 +1,10 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'capitalize',
+})
+export class CapitalizePipe implements PipeTransform {
+  transform(value: string): unknown {
```

Comment:
Won't this always return a capitalized string?

Reply:
The whole point of it is to return the string with the first letter capitalized, however i have removed the pipe since the titlecase pipe already exists which does the same